project_internal_access_from_portal/security/portal_project_rules.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <!-- Portal users see projects, if visibility = portal_internal -->
+    <record id="rule_portal_read_internal_projects" model="ir.rule">
+      <field name="name">Portal: read internal projects</field>
+      <field name="model_id" ref="project.model_project_project" />
+      <field name="domain_force">[('privacy_visibility','=','portal_internal')]</field>
```

Comment:
This rule will allow portal users to see any project with the privacy set to "Portal/internal".  Which means that user Bob will see projects of the user Jane. Is that what we want to achieve?